LiteCore/Database/PrebuiltCopier.cc

```diff
@@ -27,6 +27,32 @@
 
 namespace litecore {
     using namespace std;
+
+    static void createLocalCheckpointDocument(const unique_ptr<C4Database> &db)
+    {
+        C4UUID privateUUID;
+        C4Error err;
+        if(!c4db_getUUIDs(db.get(), nullptr, &privateUUID, &err)) {
+            error::_throw(error::Domain::LiteCore, err.code);
+        }
+
+        fleeceapi::Encoder enc;
```

Comment:
Isn't it overkill to store this in Fleece format? The public and private UUIDs are just stored as raw bytes. If you stored this one that way too, you could maybe reuse the Database methods getUUIDIfExists and generateUUID.